src/components/RightSidebar/SharedItems/SharedItemsTab.vue

```diff
@@ -47,7 +47,10 @@
 				:name="conversation.displayName"
 				:is-active="active" />
 		</template>
-		<NcEmptyContent v-else-if="!hasSharedItems">
+		<NcRelatedResourcesPanel provider-id="talk"
+			:item-id="conversation.id"
```

Comment:
I think you want the conversation.token as that is how shares with talk work?

Reply:
In my testing the https://github.com/nextcloud/related_resources app uses this integer id to correlate Talk rooms with other resources, can you confirm @ArtificialOwl?

Reply:
Right now, I am using a `room_id` from `talk_rooms1` to identify a room.

Reply:
Should really use the token, like in the oc_share table

Reply:
@ArtificialOwl is that a lot of changes ?

Reply:
raised here https://github.com/nextcloud/related_resources/issues/41
we should fix this before 25 RC1

Reply:
maybe some misunderstanding :)

I use the `room_id` only internally to get the list of rooms available to an entity.
I use the `token` to generate the url to the talk room: ex. `/index.php/call/eu2fs7w9`

Reply:
No misunderstanding here. I see that, but I'd like to see the token used in the requests to the related_resources API instead of the room_id

Reply:
ok, now I got it

Reply:
now using conversation token in backend https://github.com/nextcloud/related_resources/pull/43

Reply:
Thanks, merged and adjusted the PR here!